frame/staking-rewards/README.md

```diff
@@ -9,6 +9,7 @@ This pallet allow us to reward users for staking certain assets.
 - claiming reward at any time while the NFT has not expired
 - unstaking early, resulting in penalty applied
 - unstaking once the NFT expired
+- User owns fNFT, fNFT owns rewards. Hence selling fNFT is selling rewards to.
```

Comment:
not sure if you mean `too` instead of `to` in last.

Reply:
oh, fixed inlatest prs